libs/community/langchain_community/vectorstores/aerospike.py

```diff
@@ -155,7 +155,7 @@ def convert_distance_strategy(
     def add_texts(
         self,
         texts: Iterable[str],
-        metadatas: Optional[List[dict]] = None,
+        labels: Optional[List[dict[str, str]]] = None,
```

Comment:
As a list of dictionaries I think `metadatas` should probably stand

Reply:
The labels can not be nested and must have keys and values of type string. Like so
```
{"model": "llama3", "date": "1/1/25"}
```

Are you suggesting that I revert it?